extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfigProvider.java

```diff
@@ -25,6 +25,10 @@ public String getProperty(String key) {
                 return config.inAppPackages.map(p -> join(",", p))
                         .filter(s -> !Objects.equals(s, "*"))
                         .orElse("");
+            case DefaultSentryClientFactory.ENVIRONMENT_OPTION:
+                return config.environment.orElse("");
```

Comment:
```suggestion
                return config.environment.orElse(null);
```

Reply:
For `inAppPackages` we return `""` because it's part of the Sentry spec, but for others, `null` means undefined..